content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -519,6 +519,24 @@ section.
 
 {{% code_sample file="service/networking/tls-example-ingress.yaml" %}}
 
+{{< note >}}
+As an alternative to make TLS work on the default rule or when faced with specific 
+Subject Alternative Name requirements (e.g mandatory IPs instead of FQDN), user can update the 
+Ingress Controller to set as default certificate a custom Secret that satifies those requirements.
+
+If using Nginx Ingress:
+
+kubectl -n ingress-nginx edit deployment ingress-controller
+
+Edit the args section as follows:
+
+args:
+ - /nginx-ingress-controller
+ - '--publish-service=$(POD_NAMESPACE)/ingress-nginx-controller'
+ - '--default-ssl-certificate=default/testsecret-tls'
+ - ...
+{{< /note >}}
+
 {{< note >}}
```

Comment:
This note is useful, though - it's telling people to go and read the docs for the ingress controller(s) that they are actually using.

Reply:
The intent is to rectify or complete the note before it. See capture below.
I recently faced a mandatory mTLS requirement where the client calls an endpoint and checks if the certificate matches a given IP in the S.A.Ns. 

- If the TLS cert is attached to an ingress resource, k8s will attempt to match it through the host rule (FQDN, as IPs won't work).
- If it doesn't match, k8s will fallback to the default "Fake Certificate", which the client needs to ignore in order to connect.
- The only solution is to create a TLS cert, with the required  -ext SubjectAlternativeName:c=IP:x.x.x.x, and apply it as default cert for the ingress controller. 

Is this solution indicated in the current documentation ? @sftim 

<img width="591" alt="image" src="https://github.com/user-attachments/assets/9ad8029d-8e20-4f79-b4b1-8f41fb0b9cb0" />